### ISSUE TITLE
Cache scrobbles when offline

### ISSUE THREAD

**Issue by InputUsername** [InputUsername/rescrobbled#30]
Cache scrobbles when the user is offline. Maybe also a config option to cache scrobbles regardless of online status.

**Comment by hirad1986** [comment]
Is there any hope for having this feature implemented?

**Comment by InputUsername** [comment]
Until now it seems nobody has actually needed this feature (including myself) but I'd be happy to look into it!

**Comment by Yohe-Am** [comment]
+1 for this feature. Even better if it flushes the cache to disk. I sometimes go for many days without an internet connection.

**Comment by bonelifer** [comment]
Yes, this would be nice. Setting up a portable MPD server that will act as my MPD server at home. But also since its an old laptop on the road. Would be nice to have this feature.
